contrib/fiber/fiber_test.go

```diff
@@ -0,0 +1,242 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
We're going to need an `example_test.go` file. Please check the other integrations for guidance.

Reply:
done, thanks